hr_infraction/models/hr_employee.py

```diff
@@ -0,0 +1,13 @@
+from openerp import models, fields
+
+
+class HrEmployee(models.Model):
+
+    _name = 'hr.employee'
+    _inherit = 'hr.employee'
+    infraction_ids = fields.One2many(
+        comodel_name='hr.infraction',
```

Comment:
@feketemihai if it's not against https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md, i'd prefer keep named argument. It's more readable and easier to explain to other coder for further maintenance.

I tried to re-read https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md and i didn't find any rules regarding this. Please let me know if i'm wrong.